<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppEditor</class>
 <widget class="QDialog" name="AppEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Song Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icons.qrc">
    <normaloff>:/icons/app_icon</normaloff>:/icons/app_icon</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <item>
    <layout class="QHBoxLayout" name="CentralWidget">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="MainPanel">
       <item>
        <layout class="QHBoxLayout" name="LayoutFeedback">
         <item>
          <widget class="QLabel" name="LblFeedback">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Feedback Message</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BtnFeedback">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icons/icons.qrc">
             <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="CmbSongbooks"/>
         </item>
         <item>
          <widget class="QToolBar" name="ToolbarEditor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="windowTitle">
            <string notr="true">toolBar1</string>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <addaction name="actionSave"/>
           <addaction name="actionDelete"/>
           <addaction name="actionClear"/>
           <addaction name="actionExtra"/>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="LblNumber">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Song Number:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="TxtNumber">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="LayoutTitle">
           <item>
            <widget class="QLabel" name="LblTitle">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Title of the Song:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="TxtTitle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string>Only Believe</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LblContent">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>The rest of the Content of the Song</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="TxtContent">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLayout_3">
         <item>
          <layout class="QVBoxLayout" name="LayoutKey">
           <item>
            <widget class="QLabel" name="LblKey">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Song Key (Optional):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="TxtKey">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="LayoutAuthor">
           <item>
            <widget class="QLabel" name="LblAuthor">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Song Author (Optional):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="TxtAuthor">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="placeholderText">
              <string>Public Domain</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="LblAlias">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Optional Alias of the Song (i.e in English if not an English Song)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLayout_4">
         <item>
          <widget class="QLineEdit" name="TxtAlias">
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
   <property name="iconText">
    <string>Save Changes</string>
   </property>
   <property name="toolTip">
    <string>Save Changes</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Song</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/cancel</normaloff>:/icons/cancel</iconset>
   </property>
   <property name="text">
    <string>Clear Text</string>
   </property>
  </action>
  <action name="actionExtra">
   <property name="icon">
    <iconset resource="images/images.qrc">
     <normaloff>:/Down_Black.png</normaloff>:/Down_Black.png</iconset>
   </property>
   <property name="text">
    <string>Extra Fileds</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images/images.qrc"/>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
